vtrace/breakpoints.py

```diff
@@ -404,3 +411,17 @@ def notify(self, event, trace):
         ret_addr, args = tup
 
         self.runPostHookCallbacks(event, trace, ret_addr, args)
+
+class PosixLibLoadHookBreakpoint(Breakpoint):
+    '''
+    POSIX systems need to hook DL to identfy when libraries are loaded.
+    '''
+    def __init__(self, expression):
+        Breakpoint.__init__(self, None, expression=expression)
+        self.untouchable = True
+        self.silent = True
```

Comment:
I think setting this to self.stealthbreak would actually be more efficient here since it doesn't fire a notify break at all but still calls the callback making it act a bit closer to the intended use case?

Reply:
agreed.

Reply:
changing this.